plugins/default/lib/test_component_health.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+import unittest
+
+from component_health import parse_response
+
+class TestParseResponse(unittest.TestCase):
+    def runTest(self):
+        self.assertEqual(parse_response("{\"status\": \"ok\", \"summary\": \"Test Summary\", \"details\": []}"), (0, 'Test Summary', []))
+        self.assertEqual(parse_response("{\"status\": \"warn\", \"summary\": \"Test Summary\", \"details\": []}"), (1, 'Test Summary', []))
+        self.assertEqual(parse_response("{\"status\": \"crit\", \"summary\": \"Test Summary\", \"details\": []}"), (2, 'Test Summary', []))
```

Comment:
It they need to match nagios, the constants should not change. That's not something we can control with these tests.

At the same time/PR one would change nagios.CRIT to 42, he/she would also fix the test by changing the hard coded value to 42. So not using the constant doesn't fix the problem, don't you think?